src/module/system/text-editor.ts

```diff
@@ -513,16 +515,21 @@ class TextEditorPF2e extends TextEditor {
         const item = args.rollData?.item instanceof ItemPF2e ? args.rollData?.item : null;
         const actor = (args.rollData?.actor instanceof ActorPF2e ? args.rollData?.actor : null) ?? item?.actor ?? null;
 
-        const rollOptions = ((): string[] => {
+        const traits = ((): string[] => {
             const fromParams = params.traits?.split(",").flatMap((t) => t.trim() || []) ?? [];
             const fromItem = item?.system.traits?.value ?? [];
             return params.overrideTraits === "true" ? fromParams : R.uniq([...fromParams, ...fromItem]);
         })().sort();
+
+        const extraRollOptions = ((): string[] => params.options?.split(",").flatMap((t) => t.trim() || []) ?? [])()
```

Comment:
I can't write it as a suggestion without verifying but `R.compact(params.options?.split(",").map((t) => t.trim() ?? []).sort()` should be cleaner here.

Reply:
Should be updated now- thanks for the suggestion!